app/controllers/admin/stats_controller.rb

```diff
@@ -90,6 +90,12 @@ def polls
     @participants = ::Poll::Voter.where(poll: @polls)
   end
 
+  def gender_stats
```

Comment:
For now maybe we could remove this action (alongside its view and the entries in `admin.yml`), since admin stats are a bit different than public stats :thinking:.